src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1548,7 +1548,58 @@ to validate the workflow and continue the execution of the rest tasks; the *Mayb
 
 In order to validate your workflow using *Web Validation*, you need to go to the *ProActive Notification & Validation* portal, the workflow will be validated by clicking on one of the three buttons. By using *Email Validation*, you will receive an email with three links, the workflow can be validated by clicking on one of the links.
 
+=== Ldap Query Tasks
 
+==== Ldap query task variables table
+
+This task provides Ldap client for querying Ldap server. The query is the search query with parameters. The next parameters is provided as task variables:
```

Comment:
There are different kinds of queries in ldap. For searching, adding or removing entries.